Implement controller and repository for Board

lgusar opened the pull request:
implement controller, repository and service layer for Board
closes #16

lgusar committed:
Implement controller and repository for Board

implement controller, repository and service layer for Board
closes #16

antebm commented on projo-api/src/main/java/com/projoboard/projo/board/Board.java:
```diff
@@ -0,0 +1,70 @@
+    private String title;
+
+    @OneToMany
+    @JoinColumn(name = "board_id")
```
Update task entity with this field

lgusar replied:
The field is not added to the entity, Hibernate automatically creates the column inside the Tasks table

lgusar committed:
Implement controller and repository for Board

implement controller, repository and service layer for Board
fix tasks not deleting when the board is updated with new tasks
closes #16
